resources/dicts/conditions/pregnancy.json

```
@@ -90,7 +109,11 @@
       "m_c secretly had a {insert} with r_c.",
       "m_c hopes that {PRONOUN/m_c/poss} {insert} doesn't look too much like r_c, otherwise questions might follow.",
       "r_c goes to visit m_c in the nursery with {PRONOUN/m_c/poss} new {insert}, on a completely innocent mission to deliver food to the new parent.",
-      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. "
+      "The newly arrived {insert} that m_c has just given birth to looks suspiciously like r_c. ",
+	    "No one wanted to ruin such a happy occasion but... it didn't take a genius to notice how m_c's {insert} looked suspiciously like r_c when they shouldn't...",
+	    "m_c can see it, m_c's mate can see it, r_c can see it, everyone in c_n can see the new {insert} look too much like r_c to be coincidence...",
+        "Ever since the birth, rumor's gone around that m_c's {insert} belong to r_c's and not {PRONOUN/m_c/poss} mate and while no one has confirmed anything, no one really has to when you just look at them...",
```

```suggestion
        "Ever since the birth, rumor's gone around that m_c's {insert} belongs to r_c.  While no one has confirmed anything, no one really has to when the evidence is mewling right there...",
```
Some edits for better flow.